### PR TITLE
Address design review feedback

### PR DESCRIPTION
**Pull request by kjdchapman**
This PR will:
- Prettify paths, it's now `wifi.service.gov.uk/get-started` rather than `wifi.service.gov.uk/getstarted.html`
- Fix the styling on the 'get started' page
- Add a favicon
- Correct page titles
- Correct some small language mistakes and the call-to-action
- Linkify 'create admin account' so there''s a path from get-started to the admin app
- Add draft advice for lost end user account details

@constancecerf , would appreciate your thoughts on the last point, here's what it looks like:

image

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@
 page '/*.json', layout: false
 page '/*.txt', layout: false
 
+activate :directory_indexes
```
Comment (tommotaylor) [inline config.rb]: Nice

**Comment by tommotaylor** [comment]
Looks good to me, nice improvements.

My only feedback, which can be in separate PRs is 1) Hero doesn't look very good on mobile, 2) The "or, connecting to an existing GovWifi network" instantly struck me as weird looking. I think because the "Or" is sticking out there on its own without underlining.

screenshot 2018-10-11 at 15 01 17
screenshot 2018-10-11 at 15 03 11

**Comment by kjdchapman** [comment]
Going to have a quick look at those two points in case they're easy to fix.

**Comment by kjdchapman** [comment]
They weren't! Former I'll mention to Constance tomorrow, latter I've recorded as a thing to sort out in Icebox.

**Pull request merged**